### PR TITLE
Emulate frames left nav 'Any' search

### PR DESCRIPTION
**Pull request by sjpadgett**
- New search box does same search as frames Find By Any.
- Enter is supported in Find By Any search box.
- Search icon will open finder tab as before unless search box is populated then will open tab with 'any' results.
- All searches in finder will override 'any' results but will default back to 'any' results if all finder search boxes are reverted back to empty.
- Moved Processing... to top and changed to a refresh spinner.
- Rewrite searches sql's to support binding search param's.
- Maybe some cleanup, I forget....

below by bradymiller:
Up for Grabs demos for this PR is at: https://www.open-emr.org/wiki/index.php/Development_Demo#Delta_-_Up_For_Grabs_Demo

### CODE REVIEW

File: interface/main/tabs/js/user_data_view_model.js [hunk 1]
```diff
@@ -5,10 +5,10 @@
  * @link      http://www.open-emr.org
  * @author    Kevin Yeh <kevin.y@integralemr.com>
  * @author    Brady Miller <brady.g.miller@gmail.com>
- * @author    Ranganath Pathak <pathak@scrs1.org>
```
Comment (bradymiller) [inline interface/main/tabs/js/user_data_view_model.js]: You've removed Ranganath from existence :)

**Comment by bradymiller** [comment]
Up for Grabs demos for this PR is at: https://www.open-emr.org/wiki/index.php/Development_Demo#Delta_-_Up_For_Grabs_Demo

**Comment by bradymiller** [comment]
hi @sjpadgett ,
Very cool stuff! The only issue I noted was that Ranganath author/copyright tag was remove.
-brady

File: interface/main/tabs/js/user_data_view_model.js (hunk 1 above)
Comment (sjpadgett) [reply interface/main/tabs/js/user_data_view_model.js]: lol, well we can't have that! I guess git won't give a conflict for commented code. Maybe on merge, anyway will fix and bring in.
As a note, if you haven't tried turning on User Debug while testing, I found it very helpful while doing this PR. Ya ought to try sometime.

**Comment by sjpadgett** [comment]
Strange that the missing function changeMFA did not show up on the previous commit! I checked commits again and don't see anything else so bringing in.

**Pull request merged**